pkg/kopia/command/restore.go

```diff
@@ -24,6 +24,7 @@ type RestoreCommandArgs struct {
 func Restore(cmdArgs RestoreCommandArgs) []string {
 	args := commonArgs(cmdArgs.CommandArgs, false)
 	args = args.AppendLoggable(restoreSubCommand, cmdArgs.RootID, cmdArgs.TargetPath)
+	args = args.AppendLoggableKV(ignorePermissionsError, "false")
```

Comment:
The default can be `false` but we should make this a part of `RestoreCommandArgs`. That way, as @pchistikov said, downstream users could configure it to a different value if required